homeassistant/components/smarty/binary_sensor.py

```diff
@@ -0,0 +1,69 @@
+"""Support for Salda Smarty XP/XV Ventilation Unit Binary Sensors."""
+
+import logging
+
+from homeassistant.components.binary_sensor import BinarySensorDevice
+from . import DOMAIN
+from .sensor import SmartySensor
```

Comment:
We shouldn't rely on another platform in this platform module. Please move the common base entity class to eg the smarty component package.
```py
from . import DOMAIN, SmartySensor
```

Also, the binary sensors class needs to overwrite different properties than the sensor class. So we should make another common binary sensor base class, or remove some non common properties in the existing `SmartySensor` base class.